api/zeebe/zeebe-openapi.yaml

```diff
@@ -0,0 +1,386 @@
+      schema:
+        default: http
+        description: The schema of the Zeebe REST API server.
+
+paths:
+  /topology:
+    get:
+      tags:
+        - Cluster
+      summary: Get cluster topology
+      description: Obtains the current topology of the cluster the gateway is part of.
+      responses:
+        "200":
+          $ref: "#/components/responses/TopologyResponse"
+  /user-tasks/{userTaskKey}/completion:
+    post:
+      tags:
+        - User task
+      summary: Complete a user task
+      description: Completes a user task with the given key.
+      parameters:
+        - name: userTaskKey
+          in: path
+          required: true
+          description: The key of the user task to complete.
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserTaskCompletionRequest"
+
+      responses:
+        "204":
+          description: The user task was completed successfully.
+        "404":
+          description: The user task with the given key was not found.
+        "409":
+          description: >
+            The user task with the given key is in the wrong state currently.
+            More details are provided in the response body.
+          $ref: "#/components/responses/ProblemResponse"
+        "400":
+          description: >
+            The user task with the given key cannot be completed.
+            More details are provided in the response body.
+          $ref: "#/components/responses/ProblemResponse"
+  /user-tasks/{userTaskKey}/assignment:
+    post:
+      tags:
+        - User task
+      summary: Assign a user task
+      description: Assigns a user task with the given key to the given assignee.
+      parameters:
+        - name: userTaskKey
+          in: path
+          required: true
+          description: The key of the user task to assign.
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserTaskAssignmentRequest"
+      responses:
+        "204":
+          description: The user task's assignment was adjusted.
+        "404":
+          description: The user task with the given key was not found.
+        "409":
+          description: >
+            The user task with the given key is in the wrong state currently.
+            More details are provided in the response body.
+          $ref: "#/components/responses/ProblemResponse"
+        "400":
+          description: >
+            The assignment of the user task with the given key cannot be completed.
+            More details are provided in the response body.
+          $ref: "#/components/responses/ProblemResponse"
+  /user-tasks/{userTaskKey}/update:
```

Comment:
❌ As mentioned in the [original spec issue](https://github.com/camunda/zeebe/issues/16929#issuecomment-1999930146), this needs to be adjusted.
```suggestion
  /user-tasks/{userTaskKey}:
```

Reply:
Updated in https://github.com/camunda/camunda-docs/pull/3487/commits/216c0f1368040f5d9d97391e6e268121dc3b4cf0.